docs/guides/server/db.adoc

```diff
@@ -128,20 +128,27 @@ Then continue configuring the database as described in the next section.
 
 For each supported database, the server provides some opinionated defaults to simplify database configuration. You complete the configuration by providing some key settings such as the database host and credentials.
 
-. Start the server and set the basic options to configure a database
+The configuration can be set during a *build command* OR a *start command*:
+
+. Using a *build command* followed by an optimzed start command (recommended)
++
+----
+bin/kc.[sh|bat] build --db postgres --db-url-host mypostgres --db-username myuser --db-password change_me
```

Comment:
I know it's being done for simplicity, but should we call out that specifying the password via a cli option is not recommended?

Reply:
cf. the conf/keycloak.conf comment above for the build command.
since the build command is recommended, it's already specified